content/blog/new-graphic-features/index.Rmd

```diff
@@ -0,0 +1,301 @@
+---
+output: hugodown::hugo_document
+
+slug: new-graphic-features
+title: ragg, svglite, and the new graphics features
+date: 2022-02-25
+author: Thomas Lin Pedersen
+description: >
+    The graphics engine in R is continually improving and the ragg and svglite graphic devices 
+    follow along in supporting them. This blog post will outline what is now possible and what
+    might come in the future.
+
+photo:
+  url: https://versum.xyz/token/versum/15341
+  author: Thomas Lin Pedersen
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [deep-dive] 
+tags: [ragg, svglite, ggplot2, graphics]
+---
+
+```{=html}
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+```
+```{r, include=FALSE}
+knitr::opts_chunk$set(
+  dev = "ragg_png"
+)
+set.seed(21)
+```
+
+The release of ragg 1.2 and svglite 2.1 bought support for some exciting new graphics engine features, including gradients and patterns, that was added with R 4.1 by Paul Murrell from R Core. This post will dive into these new features as well as discuss what the future might hold for the R graphics engine.
+
+If you want to follow along on your own computer you can install the latest versions of ragg and svglite from CRAN
+
+```{r, eval = FALSE}
+install.packages("ragg")
+install.packages("svglite")
+```
+
+The post will rarely make any specific call-outs to ragg or svglite, as these are simply the packages that facilitate what is now possible with R graphics.
+
+## What is the graphics engine?
+
+You might wonder what is meant by the *R graphics engine*. It's pretty deep in the R graphics stack, so as a user, you are unlikely to ever engage with it directly, but since I somehow caught your attention we might as well indulge in the finer points of the graphics implementation.
```

Comment:
```suggestion
You might wonder what is meant by the *R graphics engine*. It's pretty deep in the R graphics stack, so as a user, you are unlikely to ever engage with it directly. But, since I somehow caught your attention, we might as well indulge in the finer points of the graphics implementation.
```